mbed_lib.json

```diff
@@ -2,7 +2,9 @@
     "name": "easy-connect",
     "target_overrides": {
         "*": {
-            "target.features_add": ["COMMON_PAL"]
+            "target.features_add": ["COMMON_PAL"],
+            "platform.stdio-baud-rate":115200,
+            "platform.stdio-convert-newlines":true
```

Comment:
Same. If we can set this for example apps and not have the \r\n issue I'm happy with updating the examples.